roots/test-default-handling/index.rst

```diff
@@ -0,0 +1,5 @@
+.. sphinx_argparse_cli::
+  :module: parser
+  :func: main
+  :no_default_values: false
```

Comment:
What's with the double negative here? Either make this not take an argument or drop the no prefix.

Reply:
Also, add a changelog entry.